varats-core/varats/mapping/author_map.py

```diff
@@ -0,0 +1,162 @@
+"""Author map module."""
+
+import logging
+import re
+import typing as tp
+from functools import reduce
+from pathlib import Path
+
+from benchbuild.utils.cmd import git, mkdir
+from plumbum import local
+
+LOG = logging.getLogger(__name__)
+
+
+class AmbiguousAuthor(Exception):
+    """Raised if an ambiguous author is encountered."""
+
+
+name_regex = re.compile("\s*\d+\t(.*) <(.*)>")
```

Comment:
```suggestion
name_regex = re.compile("\s*\d+\t(.*) <(.*)>")
```
@Sinerum can you make the constant upper case?